server/database.go

```diff
@@ -679,50 +684,50 @@ func (s *Server) registerDatabaseRoutes(g *echo.Group) {
 	})
 }
 
-func createTransferProjectActivity(ctx context.Context, stores *store.Store, database *api.Database, existingProjectID, updaterID int) error {
-	existingProject, err := stores.GetProjectV2(ctx, &store.FindProjectMessage{UID: &existingProjectID})
+func createTransferProjectActivity(ctx context.Context, stores *store.Store, database *store.DatabaseMessage, oldProject, newProject *store.ProjectMessage, updaterID int) error {
+	existingProject, err := stores.GetProjectV2(ctx, &store.FindProjectMessage{UID: &oldProject.UID})
```

Comment:
Do we need to check if the project is deleted here?

Reply:
It's checked on line 153.